doc-content/enterprise-only/installation/controller-jws-install-proc.adoc

```diff
@@ -136,36 +137,42 @@ endif::PAM[]
 ifdef::DM[]
 [source]
 ----
-set CATALINA_OPTS= -Xmx1024m 
- -Dorg.jbpm.server.ext.disabled=true 
- -Dorg.jbpm.ui.server.ext.disabled=true 
- -Dorg.jbpm.case.server.ext.disabled=true 
- -Dorg.kie.server.controller.user=<CONTROLLER_USER> 
- -Dorg.kie.server.controller.pwd=<CONTROLLER_PWD> 
+set CATALINA_OPTS= -Xmx1024m
+ -Dorg.jbpm.server.ext.disabled=true
+ -Dorg.jbpm.ui.server.ext.disabled=true
+ -Dorg.jbpm.case.server.ext.disabled=true
+ -Dorg.kie.server.controller.user=<CONTROLLER_USER>
+ -Dorg.kie.server.controller.pwd=<CONTROLLER_PWD>
  -Dorg.kie.server.id=<KIE_SERVER_ID>
  -Dorg.kie.server.location=http://<HOST>:<PORT>/kie-server/services/rest/server
- -Dorg.kie.server.controller=http://<HOST>:<PORT>/controller/rest/controller
+ -Dorg.kie.server.controller=http://<CONTROLLER_HOST>:<CONTROLLER_PORT>/controller/rest/controller
 ----
 endif::DM[]
++
+In the preceding examples, replace the following variables:
+
+* Replace `<CONTROLLER_USER>` and `<CONTROLLER_PWD>` with the user name and password for the `kie-server` role that you defined earlier in this procedure.
+* Replace `<KIE_SERVER_ID>` with a unique identifier.
+* Replace `<CONTROLLER_HOST>:<CONTROLLER_PORT>` with the host and port (IP address) of the controller. If you use the same server for {KIE_SERVER} and the controller, `<CONTROLLER_HOST>:<CONTROLLER_PORT>` is `localhost:8080`.
```

Comment:
Minor suggestion - swap positions of IP address and host and port, because we are indicating that the IP address needs the port declared.
s/with the host and port (IP address) of the controller/with the IP address (host and port) of the controller/

Reply:
Good suggestion, thanks.